.github/workflows/pytest.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     runs-on: ${{ matrix.os }}
 
     strategy:
-      fail-fast: true
+      fail-fast: false
```

Comment:
Feel free to revert this, I needed to see if the failing tests were platform specific or not.

Reply:
```suggestion
      fail-fast: true
```